conjur/init/conjurrc_data.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+
+"""
+ConjurrcData module
```

Comment:
conjurrcData is only for init? Isn't there any other usage for it outside of init?

Reply:
init is used to generate the conjurrc data. We use that conjurrc with the netrc data when we try to connect to conjur to fetch data

Reply:
So we would use it in `config.py`

Reply:
Then this file should not reside inside `init` folder.